tests/pipelines/multi_asic_201911.yml

```diff
@@ -0,0 +1,129 @@
+# Starter pipeline
+
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+
+# Add steps that build, run tests, deploy, and more:
+
+# https://aka.ms/yaml
+
+pr:
+- master
+
+name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+resources:
+  repositories:
+  - repository: sonic-mgmt
+    type: github
+    name: Azure/sonic-mgmt
+    endpoint: build
+
+stages:
+- stage: Test
+  pool: sonictest
+  variables:
+  - name: dut
+    value: vlab-08
+  - name: tbname
+    value: vms-kvm-four-asic-t1-lag
+  - name: inventory
+    value: veos_vtb
+  - name: testbed_file
+    value: vtestbed.csv
+  - name: ptf_name
+    value: ptf_test
+  - name: tbtype
+    value: multi-asic-t1-lag
+
+  jobs:
+  - job: four_asic_vs_test 
+    displayName: "kvmmultiasictest"
+    timeoutInMinutes: 0
+    steps:
+
+    - script: |
+        sudo mkdir -p /data/sonic-vm/images
+        wget https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-multiasic-vs-image-201911/lastSuccessfulBuild/artifact/target/sonic-vs.img.gz
```

Comment:
This image path can be made a variable, that can be defined at the scheduling pipeline stage.

Reply:
Modified as per suggestion.